libspawner/src/win32/runner.cpp

```diff
@@ -480,6 +506,7 @@ void runner::run_process() {
         WaitForSingleObject(process_info.hProcess, INFINITE);//may stuck here
         return;
     }
+    analyze_executable(program);
```

Comment:
> exit here if stack is too large
We never know when stack is large enough to cause `CreateProcess()` failure. We only could make an assumption that if it exceeds system guaranteed maximum, then an error may occur.
But I definitely should analyze only in case of error, sorry.